src/silo_api/api.cpp

```diff
@@ -9,16 +11,79 @@
 #include <Poco/Util/OptionSet.h>
 #include <Poco/Util/ServerApplication.h>
 #include <spdlog/spdlog.h>
+#include <boost/algorithm/string/join.hpp>
 
 #include "silo/config/config_repository.h"
 #include "silo/database.h"
 #include "silo/preprocessing/preprocessing_config.h"
 #include "silo/preprocessing/preprocessing_config_reader.h"
 #include "silo/query_engine/query_engine.h"
-#include "silo/storage/database_partition.h"
 #include "silo_api/info_handler.h"
 #include "silo_api/logging.h"
 #include "silo_api/request_handler_factory.h"
+#include "silo_api/runtime_config.h"
+
+silo::preprocessing::PreprocessingConfig preprocessingConfig(
+   const Poco::Util::AbstractConfiguration& config
+) {
+   silo::preprocessing::OptionalPreprocessingConfig default_preprocessing_config;
+   if (std::filesystem::exists("./default_preprocessing_config.yaml")) {
+      default_preprocessing_config = silo::preprocessing::PreprocessingConfigReader().readConfig(
+         "./default_preprocessing_config.yaml"
```

Comment:
Why do we both have a default preprocessing config, and then also default values in the code in case they are missing in the user provided or default preprocessing config? I do not see a case where the user would override default config values in the "default_preprocessing_config.yaml"

Reply:
We could also agree to not have default values in the code. But I would definitely give a user the opportunity to overwrite values of the default config.

Like this we have three stages:
* Default values that one can simply use (e.g. the `gene_' prefix)
* Environment specific default values (locally, you don't want to set the data path to `/data`, but in the Docker image this is convenient).
* Values that a  user likely will specify (e.g. the metadata file name).